app/lib/ui/views/active/form_field/multi_string_form_field.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:app/core/utilities/optional.dart';
+import 'package:app/ui/views/active/form_field/string_form_field.dart';
+import 'package:flutter/material.dart';
+
+/// A custom form field for List<String>
+class MultiStringFormField extends StatefulWidget {
+  final List<String> initialList;
+  final String titles;
+  final _MultiStringFormFieldState state = _MultiStringFormFieldState();
+
+  MultiStringFormField(
+      {Key key, @required this.initialList, @required this.titles})
+      : super(key: key);
+
+  @override
+  _MultiStringFormFieldState createState() => state;
+
+  List<String> getList() => state.getList();
+}
+
+class _MultiStringFormFieldState extends State<MultiStringFormField> {
+  // TODO: #134. This class should offer validation for non-empty lists, if desired
+  // Meaning that the validation function should have optional non-empty check
+  // if at least one field should exist, and the first item should have no delete button
+  final List<StringFormField> _fields = [];
+
+  List<String> getList() => _fields.map((field) => field.getString()).toList();
+
+  void _deleteField(String field) {
+    setState(() {
```

Comment:
- [ ] Oops, this doesn't work the way it should. I should key all the list items.
https://stackoverflow.com/a/59390995

Reply:
Will add video after I fix this.